src/webgpu/shader/execution/builtin/plan_float_builtin_functions.spec.ts

```diff
@@ -0,0 +1,608 @@
+export const description = `WGSL float builtin functions execution test plan`;
+
+import { makeTestGroup } from '../../../../common/framework/test_group.js';
+import { GPUTest } from '../../../gpu_test.js';
+
+export const g = makeTestGroup(GPUTest);
+
+g.test('acos,float_builtin_functions')
+  .uniqueId(0x5895c3d8ded05408)
+  .desc(
+    `
+https://www.w3.org/TR/2021/WD-WGSL-20210910/#float-builtin-functions
```

Comment:
Have we decided that we are going to go with tip-of-tree instead of the TR URL?

Reply:
To my understanding the URLs with a date a specific version and "https://www.w3.org/TR/WGSL/" is always the tip-of-tree. No?

Reply:
That sounds correct to me. (Except the TR version may occasionally be slightly behind https://gpuweb.github.io/gpuweb/wgsl/, but I don't think it matters)

Reply:
I'm using the TR URL to mark each test plan with the version of spec they came from. This is very much open to feedback.